src/jws/verify.rs

```diff
@@ -251,6 +267,59 @@ mod tests {
         }
     }
 
+    #[test]
+    fn verify_compact_works() {
+        let res = _verify_compact::<Ed25519KeyPair>(ALICE_PKEY_ED25519, ALICE_COMPACT_MSG_ED25519);
+        let res = res.expect("res is err");
+        assert_eq!(res, true);
+    }
+
+    #[test]
+    fn verify_compact_works_different_key() {
+        // use Dave's public key instead of Alice's one
+        let res =
+            _verify_compact::<Ed25519KeyPair>(CHARLIE_PKEY_ED25519, ALICE_COMPACT_MSG_ED25519);
+        let res = res.expect("res is err");
+        assert_eq!(res, false);
+    }
+
+    #[test]
+    fn verify_compact_works_changed_payload() {
+        let res = _verify_compact::<Ed25519KeyPair>(
+            ALICE_PKEY_ED25519,
+            ALICE_COMPACT_MSG_ED25519_CHANGED_PAYLOAD,
+        );
+        let res = res.expect("res is err");
```

Comment:
I would suggest add empty strings around multi-line operators. Most of the existing code follow it. Here are multiple cases.

Reply:
Added.